free-2.md

```diff
@@ -58,12 +110,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that if someone were to work on this project, perhaps only 20% of the work is spent directly engaging in the language aspects of this project (e.g., making language design decisions). This DSL is very abstracted, and demands very little of the user’s ability to code in traditional programming languages. The user is only interacting with an excel spreadsheet. 
+The bulk of the work would be spent on the "systems" aspects of the project (e.g., implementing a complicated semantics that doesn't require a lot of language design), because of the need for ----  
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think the greatest challenge is to communicate information to an existing musical notation software. It’s very easy for me to see the excel spreadsheet and write sheet music, but I have NO IDEA how to automate that process. 
+However, I know this process would involve teaching the computer about different types of chords, the structure of different intervals that compose a chord, so that I don’t have to explicitly teach the program every possible chord. 
+Example: I teach my program about the structure of major chords, minor chords, diminished chords, major 7th chords, and dominant 7th chords. Each chord is comprised of and defined by a particular set of intervals between various notes. I would also teach the program about the different inversions that a chord can be presented in. The most challenging part of the project is, the program will need to learn what it means for a certain chord to have a given root or key signature, so that it would be able to generate all the possible chords given a basic set of rules. 
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I think this could be an interesting project because it is truly a very domain-specific language and would be a useful programming language for those who can read sheet music.
+One reason this might not be a good idea is that it requires a programming language to communicate with a music composition and notation software like MuseScore. Instead of asking the program to produce sheet music with the correct durations of each chord, and for the chord to match up with lryics, perhaps a better idea would be for the user to not be concerned about the duration of the chords and their relationship to the lyrics of the song, but to just produce a set of guitar tabs with different chords, a set of chords on the same sheet?
+
```

Comment:
I think what makes your DSL especially helpful would be its ability to convert guitar tabs into sheet music, so I hope it's not too complicated working with a "notation software".  If you've figured out how to generate the chords, I feel like it would take just a small adjustment to incorporate the duration of the note.